packages/bootleg-tokens/test/BookendSharedRoyaltyToken.test.js

```diff
@@ -0,0 +1,222 @@
+const expect = require('jest-matchers');
+const BookendSharedRoyaltyToken = artifacts.require(
+  'BookendSharedRoyaltyToken'
+);
+
+contract('BookendSharedRoyaltyToken', accounts => {
+  const accountOne = accounts[0];
+  const accountTwo = accounts[1];
+  const accountThree = accounts[2];
+  const tokenId = 1;
+
+  //count is less than payment.length
+  it('count->minter->franchisor->integer', async () => {
```

Comment:
I'm not familiar with this test description notation so it's hard for me to understand what you're testing for in these blocks. Is this based on a standard?
My preference is to make the test descriptors almost like english sentences (e.g. `it('returns the default value when  count is less than payment array length', async () => {`)
While it is more verbose (and often redundant) it helps people not familiar with the code understand what the code should do. I find that often that person "not familiar with the code" is me after about a week 😄

Reply:
Yea - this was linked to test coverage if, then statements. Was mainly for my own sake. I agree its not clear. Will change if there is time. Thanks for reviewing and merging!